util/topsql/collector/cpu.go

```diff
@@ -235,6 +235,12 @@ func (s *sqlStats) tune() {
 		s.plans[""] = s.total
 		return
 	}
+	if len(s.plans) == 1 {
+		for k := range s.plans {
+			s.plans[k] = s.total
+			return
+		}
+	}
```

Comment:
Every second tune?

Reply:
Yes, this can reduce the pressure on the `repoter` behind.